shinhye/app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const dotenv = require("dotenv");
+const morgan = require('morgan');
+
+dotenv.config();
+
+app.use(cors());
+app.use(morgan('combined'));
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
```

Comment:
myDataSource와 같이 my라는 표현은 지양하는 것이 좋습니다.
서버를 가리키는지? 코드를 작성한 개발자를 가리키는지? 아니면 ... 이 어플리케이션 자체를 가리키는지?

그렇기 때문에 다음과 같이 수정할 수 있습니다.
```suggestion
const dataSource = new DataSource({
// const appDataSource = new DataSource({
// const mysqlDataSource = new DataSource({
```

Reply:
확인 후 반영했습니다 감사합니다!